src/js/post-911-gib-status/components/EnrollmentPeriod.jsx

```diff
@@ -79,7 +76,7 @@ class EnrollmentPeriod extends React.Component {
     return (
       <div>
         <hr/>
-        <h4>{formatDateShort(enrollment.beginDate)} to {formatDateShort(enrollment.endDate)} at <span className="facility">{enrollment.facilityName.toLowerCase()}</span> ({enrollment.facilityCode})</h4>
+        <h4>{formatDateShort(enrollment.beginDate)} to {formatDateShort(enrollment.endDate)} at <span className="facility">{(enrollment.facilityName || '').toLowerCase()}</span> ({enrollment.facilityCode})</h4>
```

Comment:
Something I've been wondering about is if we should be checking if data exists before rendering anything. I need to double check the swagger yaml, to see which fields are optional from EVSS, because it would be weird if we didn't have `facilityName` or `facilityCode` for some reason, but we're still rendering this whole line. So it would be `06/15/17 to 6/15/18 at` for this line. Should we be checking if all these fields exist first? Is there a clean way of doing that?